client/src/navigator/Stacks/Stacks.js

```diff
@@ -62,12 +63,19 @@ More reading: https://reactnavigation.org/docs/stack-navigator/
 
 export const AuthNavigator = () => (
   <AuthStack.Navigator
-    initialRouteName="Landing"
+    initialRouteName="SelectLanguage"
     headerMode="screen"
     screenOptions={{
       headerShown: false,
     }}
   >
```

Comment:
Do we ensure that this screen is only shown when the user opens the app for the first time?

Reply:
Yes, once logged in this page doesn't show up.